lib/utf8_sanitizer.rb

```diff
@@ -22,7 +24,23 @@ def call(env)
 
   private
 
-  def invalid_strings(values)
+  # @param [Hash] params
+  def params_have_null_byte?(params)
+    params.values.any? { |value| contains_null_byte?(value) }
```

Comment:
Out of curiosity, do we care about keys having null bytes?

Reply:
It's probably not a good thing.... will update to check for those

Reply:
updated in f8030a78b4fa91cd86fe7736494ecd7ccee75ff3